windows/src/developer/TIKE/compile/CompileKeymanWeb.pas

```diff
@@ -2213,8 +2213,9 @@ function TCompileKeymanWeb.JavaScript_SetupProlog: string;
 begin
   if IsKeyboardVersion10OrLater then
   begin
-    Result := 'if(parseFloat(tavultesoft.keymanweb.version) < 10) {'+nl+
-      FTabStop+'tavultesoft.keymanweb.util.alert("This keyboard requires KeymanWeb 10.0 or later");'+nl+
+    Result := 'if(typeof keyman === ''undefined'') {'+nl+
+      FTabStop+'console.log(''Keyboard requires KeymanWeb 10.0 or later'');'+nl+
+      FTabStop+'if(typeof tavultesoft !== ''undefined'') tavultesoft.keymanweb.util.alert("This keyboard requires KeymanWeb 10.0 or later");'+nl+
```

Comment:
My one question:  where does this block of code get placed?  Before the `KR`/`registerKeyboard` call, in `gs()`...?

Reply:
This gets placed at the very start of the file, for example:
```
if(typeof keyman === 'undefined') {
  console.log('Keyboard requires KeymanWeb 10.0 or later');
  if(typeof tavultesoft !== 'undefined') tavultesoft.keymanweb.util.alert("This keyboard requires KeymanWeb 10.0 or later");
} else {
KeymanWeb.KR(new Keyboard_version10chiral());
}
function Keyboard_version10chiral()
{
```